website/app/templates/head.hbs

```diff
@@ -37,9 +37,7 @@ We've made some small customizations to simplify this compared to the default te
 
 {{#if this.model.imgSrc}}
   <meta property="og:image" content={{this.model.imgSrc}} />
-  <meta property="og:image:type" content="image/png" />
-  <meta property="og:image:width" content="256" />
```

Comment:
it turns out these didn't seem to matter and would be problematic anyhow since the dimensions of this image depend on which type we're sharing, and AFACT there is no way to change these dynamically

Reply:
👍